pkg/storage/store_test.go

```diff
@@ -373,7 +423,7 @@ func Test_store_GetSeries(t *testing.T) {
 				{Labels: mustParseLabels("{foo=\"bar\"}")},
 				{Labels: mustParseLabels("{foo=\"bazz\"}")},
 			},
-			1,
+			2,
```

Comment:
? why do you need to change that ?

Reply:
Not required. Reverted my change. Changed logic here https://github.com/grafana/loki/blob/30a0657e0e7e01617dc14ccd7bb78eae86d4678e/pkg/storage/util_test.go#L209

Reply:
Still this is a hacky thing. However, I just want to keep this because this is kind of complete testing for chunk along with stream